quickstart/kubernetes/index.md

```diff
@@ -5,79 +5,155 @@ redirect_from: /reference/kubernetes.html
-   the rich status updates Pulumi offers, allowing you to reliably and reproducibly deploy code,
-   even if it isn't yours.
-* [The canonical Kubernetes Guestbook example](./tutorial-guestbook.html). You can see the original
-   example [here](https://github.com/pulumi/examples/tree/master/kubernetes-ts-guestbook) for
-   comparison.
-* [Triggering a new deployment when changing the data in a
-   `ConfigMap`](./tutorial-configmap-rollout.html).
-   This example shows how Pulumi will safely allow you to update containers to use new configuration
-   data in a way that is natural to the user.
-* [Staged rollout gated on Prometheus health checks](./tutorial-p8s-rollout.html). This example
-  shows how we can roll out to a small canary ring, check Prometheus metrics, and then roll out to a
-  larger prod ring only when that succeeds.
-* [Run a Stateless App Deployment](./tutorial-stateless-app.html): Deploy a simple stateless Nginx web server backed
-    by a Docker container
+[Kubernetes][k8s] is an open source platform for running containerized applications on a cluster of
+machines. Applications are managed through a RESTful API that exposes mechanisms to deploy, scale,
+and introspect on resources in the cluster.
+
+Pulumi is an infrastructure-as-code tool that exposes a Kubernetes SDK that allows users to write
+Kubernetes applications in the language of their choice, such as JavaScript and Python. It is
+designed to fit in anywhere you already use Kubernetes:
+
+-   **API-compatible with Kubernetes** -- users do **not** have to learn a new API to write
+    Kubernetes applications with Pulumi. (See the [Kubernetes][api-reference] and
+    [Pulumi](/reference/pkg/nodejs/@pulumi/kubernetes/index.html) API documentation for more
+    details.)
+-   **Compatible with Kubernetes YAML and Helm Charts.** We will see examples of this in the
+    subsequent sections.
+-   **Drop-in replacement for `kubectl`.** Anywhere you have a [kubeconfig file][kubeconfig], you
+    can use Pulumi.
+
+The remainder of this document will demonstrate how to use Pulumi for Kubernetes development, as
+illustrated through a series of use cases and tutorials.
+
+> **Note:** Pulumi supports any Kubernetes version that is currently supported by Kubernetes
+> upstream. Typically a Kubernetes minor release is supported by upstream for a 9 month window, with
+> all supported versions being listed in [the reference docs][reference]. If you have a more
+> specific version requirement, please contact us.
+
+## Pulumi Features we Will See
+
+In addition to being compatible with all existing Kubernetes workflows, Pulumi provides several
+advantages over bare `kubectl`. In the course of these tutorials, we will see:
+
+-   **Proactive reports of errors if resource initialization fails.** If Pulumi understands why a
+    resource failed to become healthy, it will tell you at provisioning time -- no more 20 questions
+    with `kubectl`.
```

Comment:
nit: [tilt](https://tilt.build) uses this same "stop playing 20 questions" terminology. any great alternatives?

Reply:
Oh lol I thought I got that from you. Oops.